index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header class="wrapper">
+        <nav class="navbar">
+           <p class="nav-left">Joe Doe</p>
+           <div class="hamberg">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+           </div>
+
+        </nav>
+
+        <div class="header-section">
+            <h1>Laura Jong</h1>
+            <h2>Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience!
+                 If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+        </div>
+
+        <div class="eclipse"></div>
+    </header>
```

Comment:
- [ ] Kindly make sure that only the `nav` is wrapped inside the `header` tag and wrap the intro section in a `section` tag, you can use something like the following:
```
<header>
   <nav>
      ...
   </nav>
</header>
<section>
   ...
</section>
```